var/spack/repos/builtin/packages/form/package.py

```diff
@@ -10,22 +10,19 @@ class Form(AutotoolsPackage):
     """FORM is a Symbolic Manipulation System."""
 
     homepage = "https://www.nikhef.nl/~form/"
-    url      = "https://github.com/vermaseren/form/archive/v4.2.1.tar.gz"
+    url      = "https://github.com/vermaseren/form/releases/download/v4.2.1/form-4.2.1.tar.gz"
     maintainers = ['iarspider']
```

Comment:
```suggestion
    maintainers = ['iarspider', 'tueda']
```
@iarspider I guess this also meant for this? (you can apply my suggestion with the button below for it)

Reply:
Sure.

Reply:
Yes - both adding tueda as a maintainer and changing gmp and zlib defaults

Reply:
Done, that's all from me as well! :+1: Thanks!

Reply:
Oh, sorry, I pushed the button without reading it carefully...